ev-dev-tools/src/ev_cli/templates/interface-Impl.hpp.j2

```diff
@@ -49,6 +49,7 @@ private:
 
     virtual void init() override;
     virtual void ready() override;
+    virtual void shutdown() override;
```

Comment:
I just learned that virtual and override is redundant, one only ever needs to use either, `virtual`, `override` xor `final`
In our case `final` would be appropriate